.github/workflows/cicd-pipeline.yaml

```diff
@@ -4,7 +4,7 @@ on:
   push:
   pull_request:
     branches:
-      - main
+      - task/DR-391_Unit_test_action
```

Comment:
Are you going to refactor this stuff into our new develop workflow once my PR is merged?